src/react-apps/applications/shared/src/navigation/main-header/appBarConfig.tsx

```diff
@@ -0,0 +1,27 @@
+export const menu = ([
+  {
+    key: 'Om',
+    activeSubHeaderSelection: 'about',
+    link: '/aboutservice',
+  },
+  {
+    key: 'Lage',
+    activeSubHeaderSelection: 'create',
+    link: '/create',
+  },
+  {
+    key: 'Språk',
```

Comment:
Should use text from language api

Reply:
All hard coded texts will be fixed in separate issue.